src/triage/component/catwalk/utils.py

```diff
@@ -61,10 +62,10 @@ def retry_if_db_error(exception):
 
 
 @db_retry
-def save_experiment_and_get_hash(config, db_engine):
+def save_experiment_and_get_hash(config, random_seed, db_engine):
```

Comment:
Does the random seed not get passed as part of the config here? It feels slightly awkward to me to have this included as a separate parameter.

Reply:
I wanted to be explicit that *it is not part* of the definition of an `experiment`.

Reply:
Ah, ok -- I see. I think that's fine for now, though I wonder if down the road we might want to pass something like `config` and `additional_parameters` (or something like that) if we end up adding other configuration parameters that we don't want as part of the experiment definition/hash.